docs/intentReaderHandlers/RegisterProviderHandler.md

```diff
@@ -0,0 +1,187 @@
+# RegisterProviderHandler 
+
+## Overview
+
+This handler serves the purpose of simulating user inputs in FCA. Specific device calls prompt the system to request user input. When utilized alongside test providers found in src/providers and the SetApiResponseHandler, you gain the ability to provide mock responses to the device's provider calls. This is useful for manipulating various provider APIs such as keyboard and profile calls. Test provider will manage any provider that has refui. 
+
+It performs the following actions:
+1. Parses the input message received to get the provider.
+2. Informs FCA to use testProviders for setting the responses.
+3. Formats and sends the response back to the IntentReader.
+
+## Usage
+
+#### Use case - Requirement is to return abc@gmail.com for keyboard.email api
```

Comment:
There are extra bullet points. Would it be possible to have those removed and add it under the corresponding point itself ?
![image](https://github.com/rdkcentral/firebolt-certification-app/assets/148241508/0725708a-46e6-4169-a63d-dd416fb93d07)